core/store/src/config.rs

```diff
@@ -190,20 +186,22 @@ impl Default for StoreConfig {
             // we use it since then.
             block_size: bytesize::ByteSize::kib(16),
 
-            // deprecated
-            trie_cache_capacities: vec![],
-
             trie_cache: TrieCacheConfig {
                 default_max_bytes: DEFAULT_SHARD_CACHE_TOTAL_SIZE_LIMIT,
                 // Temporary solution to make contracts with heavy trie access
                 // patterns on shard 3 more stable. It was chosen by the estimation
                 // of the largest contract storage size we are aware as of 23/08/2022.
                 // Consider removing after implementing flat storage. (#7327)
+                // Note: 1_000_000_000 is correct size for RPC node, use 3_000_000_000
+                // for archival node.
                 per_shard_max_bytes: HashMap::from_iter([(
                     ShardUId { version: 1, shard_id: 3 },
-                    3_000_000_000,
+                    1_000_000_000,
```

Comment:
Hm, can you please coordinate with the node team? I think they were working under the opposite assumption regarding the default cache size.
See https://pagodaplatform.atlassian.net/browse/ND-360

Reply:
Yeah, this is more reasonable. I forgot about the issue and the thread